core/server/data/migrations/versions/3.0/3-remove-page-column.js

```diff
@@ -0,0 +1,42 @@
+const common = require('../../../../lib/common');
+const commands = require('../../../schema').commands;
+
+const createLog = type => msg => common.logging[type](msg);
+
+function createColumnMigration({table, column, dbIsInCorrectState, operation}) {
+    return function columnMigrations(options) {
+        return options.transacting.schema.hasColumn(table, column)
+            .then(dbIsInCorrectState)
+            .then((isInCorrectState) => {
+                const log = createLog(isInCorrectState ? 'warn' : 'info');
+
+                log(`Adding ${column} column to ${table} table`);
```

Comment:
Same issue here as in [1-add-type-column.js](https://github.com/TryGhost/Ghost/pull/10944/commits/516e92c89e6472db4af17b7cd09684bac3034564#r307211937)